app/models/season/phase_switcher.rb

```diff
@@ -50,4 +61,12 @@ def self.back_to_reality
     })
   end
 
+  def self.whitelisted_phases
```

Comment:
Since you named the memoized class variable `phases`, I would name the method to match that